src/api/v4/dataview/base.js

```diff
@@ -115,6 +117,52 @@ Base.prototype.getColumn = function () {
   return this._column;
 };
 
+/**
+ * Add a filter
+ *
+ * @param  {carto.filter.Base} filter
+ * @return {carto.dataview.Base} this
+ * @api
+ */
+Base.prototype.addFilter = function (filter) {
+  this._checkFilter(filter);
+  if ((filter instanceof BoundingBoxLeafletFilter) && (filter !== this._boundingBoxFilter)) {
+    this._boundingBoxFilter = filter;
```

Comment:
should we create a private _addBoundingBoxFilter to encapsulate this code?

Reply:
Yep